stories/core/SkipToContentLink.stories.tsx

```diff
@@ -0,0 +1,191 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import {
+  SkipToContentLink,
+  SkipToContentLinkProps,
+  SkipToContentMarker,
+  Header,
+  HeaderLogo,
+  HeaderBreadcrumbs,
+  HeaderButton,
+  Body,
+  Kbd,
+  Anchor,
+} from '../../src/core';
+import { CreeveyStoryParams } from 'creevey';
+
+export default {
+  component: SkipToContentLink,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Core/SkipToContentLink',
+  parameters: {
+    creevey: {
+      captureElement: null,
+      tests: {
+        async open() {
+          const body = await this.browser.findElement({ css: '.iui-body' });
+          await body.sendKeys(this.keys.TAB);
+          const opened = await this.takeScreenshot();
+          await this.expect({ opened }).to.matchImages();
+        },
+      },
+    } as CreeveyStoryParams,
+  },
+} as Meta<SkipToContentLinkProps>;
+
+export const Basic: Story<SkipToContentLinkProps> = () => {
+  React.useEffect(() => {
+    window.frameElement?.setAttribute('Name', 'storybook-iframe');
```

Comment:
Maybe add a comment explaining what it does / why it's needed

Reply:
Not needed; artifact from previous work.